src/sql/parts/dashboard/common/dashboardPage.component.html

```diff
@@ -12,9 +12,9 @@
 		</div>
 		<panel style="flex: 1 1 auto; position: relative" class="dashboard-panel" (onTabChange)="handleTabChange($event)" (onTabClose)="handleTabClose($event)" [actions]="panelActions">
 			<tab *ngFor="let tab of tabs" [title]="tab.title" class="fullsize" [identifier]="tab.id" [canClose]="tab.canClose" [actions]="tab.actions">
-				<dashboard-webview-tab *ngIf="tab.isWebview" [tab]="tab">
+				<dashboard-webview-tab *ngIf="isWebviewContentType(tab)" [tab]="tab">
```

Comment:
you may want to come up with a more flexible way to determine what to show since I think we'll need additional types of tabs beyond these two